components/admin/permit-holders/DoctorInformationCard.tsx

```diff
@@ -11,7 +11,11 @@ type DoctorInformationProps = {
   readonly previousPhysicianData: PreviousPhysicianData[];
   readonly onSave: (physicianData: UpsertPhysicianInput) => void;
 };
-
+/**
```

Comment:
```suggestion

/**
```

Reply:
Also ensure that this component is still functional in the individual permit holder page